store/driver/tikv_driver.go

```diff
@@ -142,7 +143,7 @@ func (d TiKVDriver) OpenWithOptions(path string, options ...Option) (kv.Storage,
 	}
 
 	// FIXME: uuid will be a very long and ugly string, simplify it.
-	uuid := fmt.Sprintf("tikv-%v", pdCli.GetClusterID(context.TODO()))
+	uuid := fmt.Sprintf("tikv-%v-%s", pdCli.GetClusterID(context.TODO()), keyspaceName)
```

Comment:
Should we notice the UUID  without ```keyspaceName``` in the API V1.

Reply:
it's redundant code,I'll remove the `keyspaceName`